docs/core/docker/intro-net-docker.md

```diff
@@ -33,7 +33,7 @@ The [relationship between Docker containers, images, and registries](../../stand
 
 * [Windows-based containers: Modern app development with enterprise-grade control.](https://www.youtube.com/watch?v=Ryx3o0rD5lY&feature=youtu.be)
 * [Docker overview](https://docs.docker.com/engine/docker-overview/)
-* [Dockerfile on Windows Containers](/virtualization/windowscontainers/manage-docker/manage-windows-dockerfile.md)
+* [Dockerfile on Windows Containers](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/manage-docker/manage-windows-dockerfile.md)
```

Comment:
the issue with that link was actually the .md extension in there. Can you please the link from `/virtualization/windowscontainers/manage-docker/manage-windows-dockerfile.md` to just `/virtualization/windowscontainers/manage-docker/manage-windows-dockerfile` instead?

Reply:
Ok I am sending PR again with requested changes

Reply:
Just to make sure, you don't need to create a new PR. Just make a new commit on the same branch and the changes will show up here. Thanks for contributing to docs!

Reply:
[Correct Link](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/manage-docker/manage-windows-dockerfile.md)

Reply:
The problem is this file lives in  [Virtualization-Documentation Repository](https://github.com/MicrosoftDocs/Virtualization-Documentation)

Reply:
It will work on the built topic. This is how we do cross-repo links. The build process will add https://docs.microsoft.com/locale in front of it.

If you look at the topic on docs (https://docs.microsoft.com/en-us/dotnet/core/docker/intro-net-docker), you'll see that the broken link looks like this: https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/manage-windows-dockerfile.md. 

If you remove .md from it, it'll start working.